tests/com.b2international.snowowl.test.dependencies/.classpath

```diff
@@ -2,6 +2,8 @@
 <classpath>
 	<classpathentry kind="src" path="src"/>
 	<classpathentry kind="src" path="xtend-gen"/>
+	<classpathentry exported="true" kind="lib" path="system-rules-1.18.0-sources.jar"/>
```

Comment:
This should be added as `sourcepath` to the existing `system-rules` classpath entry.